contracts/feirari/RewardsDistributorAdmin.sol

```diff
@@ -1,20 +1,21 @@
 pragma solidity ^0.8.0;
 
-import "./TribalChief.sol";
 import "../refs/CoreRef.sol";
-import "../external/rari/IRewardsDistributor.sol";
+import "./IRewardsDistributorAdmin.sol";
 
-contract RewardsDistributorAdmin is IRewardsDistributor, CoreRef {
-    /// rewards distributor contract address
-    IRewardsDistributor public rewardsDistributorContract;
+contract RewardsDistributorAdmin is IRewardsDistributorAdmin, CoreRef {
+    /// @notice rewards distributor contract address
+    IRewardsDistributorAdmin public rewardsDistributorContract;
 
+    /// @param coreAddress address of core contract
+    /// @param _rewardsDistributorContract address of the admin rewards distributor contract
     constructor(
         address coreAddress,
-        IRewardsDistributor _rewardsDistributorContract
+        IRewardsDistributorAdmin _rewardsDistributorContract
     ) CoreRef(coreAddress) {
         rewardsDistributorContract = _rewardsDistributorContract;
-        // Create new admin role for this contract
-        // grant this role to all AutoRewardsDistributors so that they can call 
+        /// @notice Create new admin role for this contract
+        /// grant this role to all AutoRewardsDistributors so that they can call 
```

Comment:
Is this comment a directive or a description? Should be a directive i.e. "governance needs to ..."

Reply:
This is a directive. Fixed.